modules/ROOT/pages/vpn-about.adoc

```diff
@@ -0,0 +1,70 @@
+= Anypoint VPN
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Anypoint VPN feature enables you to create a secure connection to systems within your network that are hosted outside of Anypoint Platform. If required, you can create multiple VPNs to connect to different networks or different regions within a network.
+
+[NOTE]
+The number of VPNs you can create depends on the VPN capacity allocated to your account.
+
+To create and maintain a secure connection with your external network, the Anypoint VPN provides the following features:
+
+* Route-based access via 2-byte and 4-byte Autonomous System Numbers (ASN). This defines the routing policies used to connect to your network through the VPN.
```

Comment:
Why capitalize Autonomous System Numbers?

Reply:
Why not? HAHAHAA!!